blog post - why we're changing to Nail Self Serve (#3505)
PostHog/posthog.com: contents/blog/changing-to-self-serve.md

---
date: 2022-05-19

> title: Next Up - Nailing Self Serve

Reviewer: Trying to think of title that doesn't involve the word 'pivot'.

Probably something like:

- Why 'Nailing Self Serve' is PostHog's new focus
- We're focused on 'Nailing Self Serve'. Here's why

Author: I think these suggestions both sound like the company pivoted, whereas we just clarified what we need to tackle next.

I'm not really writing this with HN in mind,  it's more a story for fans - I prefer mine at the moment but I'm open to more suggestions if you have any (I don't)

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["CEO diaries", "Inside PostHog"]
author: ["james-hawkins"]
featuredImage: ../images/blog/ceo-diary.png
featuredImageType: full
---

The longer your strategy gets, the more useless it becomes.

> At PostHog, we therefore communicate strategy as 'Nail X', where X is the thing we need to nail. It's simple and effective.

Reviewer: ```suggestion
That's why, at PostHog, we communicate strategy as 'Nail X', where X is the thing we need to nail. It's simple and effective.
```

We [recently switched](https://github.com/PostHog/posthog.com/pull/3395) from Nail Enterprise to Nail Self Serve. This walks you through the previous context, and why we made that change.

## Why did we focus on Nail Enterprise in the first place?

The best products are generally built for end users.

The game we're playing isn't to please C-level executives or procurement - we want an engineer in any size organization on Earth to be able to use our software in production without getting up from their desk. If they work at a larger company, and if we provide them with a great experience, they'll share the platform with their peers, and will probably want to upgrade to one of our [paid offerings](/pricing). Our open source and free user base is our sales team.

Our original go to market plan was therefore a little like this:

1. get our open source project to product market fit
1. get a paid mid-size (at the time, ~$20K-$40K a year) product to product market fit

> 1. get a paid large-size ($200K+ a year) product to product market fit

Reviewer: Assume this is meant to be a 1,2,3 list, or is this deliberate?

Author: this should turn into a 1,2,3 list automatically - check the preview

> It took us until last Summer to get the open source product to feel successful. It became a lot less buggy, easier to deploy, much fuller featured and more scalable. As a result, we saw our inbound growth increasing and our retention improving.

Reviewer: ```suggestion
It took us until last Summer to get the open source product to feel successful. It became less buggy, easier to deploy, much fuller featured and more scalable. As a result, we saw our inbound growth increasing and our retention improving.
```

> We then focused on getting 5 reference customers. It turned out, we had to [Nail Funnels](new-vp-nailing-funnels) to do this.

Reviewer: ```suggestion
We then focused on getting five reference customers. It turned out, we had to [Nail Funnels](new-vp-nailing-funnels) to do this.
```

## What changed?

> Things went much better than we thought. For the last 6 months, our paid products have averaged 20% month over month revenue growth - that's an 8.9x increase per year (for context - [3x is considered great](https://twitter.com/nimaroohis/status/1525970172237991939?s=21&t=qzYbxjpPbHlPZ7Xjus_PUA), and [Lenny's Newsletter breaks this out in more detail](https://www.lennysnewsletter.com/p/what-is-a-good-growth-rate?s=r)). 

Reviewer: ```suggestion
Things went much better than we thought. For the last six months, our paid products have averaged 20% month over month revenue growth - that's an 8.9x increase per year (for context - [3x is considered great](https://twitter.com/nimaroohis/status/1525970172237991939?s=21&t=qzYbxjpPbHlPZ7Xjus_PUA), and [Lenny's Newsletter breaks this out in more detail](https://www.lennysnewsletter.com/p/what-is-a-good-growth-rate?s=r)). 
```

Three things combined caused this:

* we were able to get larger (generally, up to $70k) deals than we expected
* we made it easier to _pay_ for our paid product, which [led to more deals](pricing-lessons) that we expected
* we had more existing customer growth than we expected

Then we noticed something else.

> Some customers are very fast (a couple of weeks, or less) to sell to, with minimal interaction, whereas others take months and are very intense. In fact, almost none of the customers that have put us through a big buying process have in fact converted to revenue yet.

Reviewer: ```suggestion
Some customers are very fast (a couple of weeks, or less) to sell to, with minimal interaction, whereas others take months and are very intense. In fact, almost none of the customers that have put us through a big buying process have converted to revenue yet.
```

## What does Nail Self Serve mean?

So we should do more of what's working.

For product, this means:

> * Quality: Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch

Reviewer: ```suggestion
* **Quality:** Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
```

> * Quality: Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
> * Self Service Subscription: Customers of every size sign-up, start using and subscribe to PostHog without asking us for help
> * Deployments: PostHog is the easiest self-hosted product to deploy and scale in the world

Reviewer: ```suggestion
* **Quality:** Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
* **Self-Service Subscription:** Customers of every size sign-up, start using and [subscribe to PostHog without asking us for help](/blog/transparent-enterprise-pricing)
* **Deployments:** PostHog is the easiest self-hosted product to deploy and scale in the world
```

For go to market, this means:

* Customer Success: Focus on time to revenue (for example, one of our Customer Success team's key results is volume of deals in our target range per week - rather than trying to hit an overall revenue figure)
* Marketing: Accelerate word of mouth growth
* Website and Docs: Make community self serve (for example, [make our _docs_](https://squeak.posthog.com) rather than Slack the main support channel)

> ## A snapshot of the view future

Reviewer: ```suggestion
## A snapshot of the future
```

> Today, we're very good at serving large mid market B2B companies who can buy quickly (through self serving).

Reviewer: ```suggestion
Today, we're very good at serving large mid-market B2B companies who can buy quickly (through self serving).
```

I suspect we'll focus on B2C after the above is completed. From customer calls, it feels like this is how we'll go from landing $20k-$70k customers in a couple of weeks to $100k to $300k customers in the same, short time frame. It's much _more_ appealing to do this, than to sell to B2B Enterprises that'll take months or years to purchase from us.

We already have plenty of (paying) customers in B2C, but to focus on it, would require stronger mobile support, greater scalability still and a suitable pricing model so we capture the right amount of value.

> Long term, we think the future of our revenue will come from large enterprise. We're a natural fit - data control, consolidating multiple vendors, and complete extensibility. However, we want to avoid _needing_ to go through complex purchasing decisions whilst there are easier ways to increase our numbers.

Reviewer: ```suggestion
Long term, we think the future of our revenue will come from large enterprise. We're a natural fit - data control, consolidating multiple vendors, and complete extensibility. However, we want to avoid _needing_ to go through complex purchasing decisions whilst there are easier ways to increase our numbers.

_Did you enjoy this article? Get more like it in your inbox every two weeks_

 <NewsletterForm
compact
/>
```

---

> date: 2022-05-19

Reviewer: ```suggestion
date: 2022-06-09
```

> title: Next Up - Nailing Self Serve

Reviewer: ```suggestion
title: Our new objective: Nail Self Serve
```

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
categories: ["CEO diaries", "Inside PostHog"]
author: ["james-hawkins"]
featuredImage: ../images/blog/posthog-ceo-diary-blog.png
featuredImageType: full
---

The longer your strategy gets, the more useless it becomes.

That's why, at PostHog, we communicate strategy as 'Nail X', where X is the thing we need to nail. It's simple and effective.

We [recently switched](https://github.com/PostHog/posthog.com/pull/3395) from Nail Enterprise to Nail Self Serve. This walks you through the previous context, and why we made that change.

> 

Reviewer: ```suggestion
>James Hawkins is PostHog's Co-Founder & CEO. He launched PostHog with Tim Glaser in February 2020 as part the YCombinator W20 batch. He lives in Cambridge, England with his wife Fran, daughter Ruby, and Wally (a cat).
```
